navigation/ca_move_base/launch/navigate.launch

```diff
@@ -5,14 +5,19 @@
 	<include file="$(find ca_bumper2pc)/launch/standalone.launch"/>
 
 	<!-- move_base -->
-	<include file="$(find ca_move_base)/launch/move_base.launch">
+	<include if="$(eval str(arg('localization')) != 'slam')" file="$(find ca_move_base)/launch/move_base.launch">
     <arg name="ns" value="$(arg robot_name)"/>
   </include>
 
 	<!-- AMCL -->
   <include if="$(eval str(arg('localization')) == 'amcl')" file="$(find ca_move_base)/launch/amcl.launch">
     <arg name="ns" value="$(arg robot_name)"/>
   </include>
+  
+  <node name="map_static_tf1" pkg="tf" type="static_transform_publisher" 
```

Comment:
These nodes should be moved to the multi-robot launch file and explicitly set `LOCALIZATION=asd`.